tools/run_tests/run_tests_matrix.py

```diff
@@ -180,15 +180,6 @@ def _create_test_jobs(extra_args=[], inner_jobs=_DEFAULT_INNER_JOBS):
                                 ['--report_multi_target'],
                                 inner_jobs=inner_jobs)
 
-    # supported on linux only
```

Comment:
Maybe we want to keep this? This is on `php7`.

Reply:
It's included later at https://github.com/HannahShiSFB/grpc/blob/60b5d06ec1c069e96dc7163aad3412754716ff56/tools/run_tests/run_tests_matrix.py#L233